front/components/assistant_builder/ActionScreen.tsx

```diff
@@ -547,80 +548,13 @@ export default function ActionScreen({
-            <span className="font-semibold">
-              It’s the best approach with large quantities of data.
-            </span>
-          </div>
-          <div className="grid grid-cols-2 gap-4">
-            <div>
-              <p>
-                <strong>Select your sources with care</strong> The quality of
-                the answers to specific questions will depend on the quality of
-                the data.
-              </p>
-              <p className="mt-1">
-                <strong>
-                  You can narrow your search on most recent documents
-                </strong>{" "}
-                by adding instructions in your prompt such as 'Only search in
-                documents from the last 3 months', 'Only look at data from the
-                last 2 days', etc.
-              </p>
-            </div>
-            <div>
-              <p>
-                <strong>Note:</strong> The available data sources are managed by
-                administrators.
-              </p>
-            </div>
-          </div>
-
-          <DataSourceSelectionSection
-            dataSourceConfigurations={builderState.dataSourceConfigurations}
-            openDataSourceModal={() => {
-              setShowDataSourcesModal(true);
-            }}
-            canAddDataSource={configurableDataSources.length > 0}
-            onManageDataSource={(name) => {
-              setDataSourceToManage(
-                builderState.dataSourceConfigurations[name]
-              );
-              setShowDataSourcesModal(true);
-            }}
-            onDelete={deleteDataSource}
-          />
-        </ActionModeSection>
-
-        <ActionModeSection show={builderState.actionMode === "DUST_APP_RUN"}>
-          <div className="text-sm text-element-700">
-            The assistant will execute a Dust Application of your design before
-            answering. The output of the app (last block) is injected in context
-            for the model to generate an answer. The inputs of the app will be
-            automatically generated from the context of the conversation based
-            on the descriptions you provided in the application's input block
-            dataset schema.
-          </div>
-          <DustAppSelectionSection
-            show={builderState.actionMode === "DUST_APP_RUN"}
-            dustAppConfiguration={builderState.dustAppConfiguration}
-            openDustAppModal={() => {
-              setShowDustAppsModal(true);
-            }}
-            onDelete={deleteDustApp}
-            canSelectDustApp={dustApps.length !== 0}
-          />
-        </ActionModeSection>
-
-        <ActionModeSection show={builderState.actionMode === "TABLES_QUERY"}>
           <div className="text-sm text-element-700">
             The assistant will generate a SQL query from your request, execute
             it on the tables selected and use the results to generate an answer.
           </div>
+
```

Comment:
Copy above is not in the figma. I think it's worth keeping it :) but just mentioning